include/seqan/basic/basic_type.h

```diff
@@ -499,7 +499,7 @@ _referenceCast(typename RemovePointer<T>::Type & ptr)
 template <typename T>
 struct LENGTH
 {
-    enum { VALUE = 1 };
+    static const unsigned VALUE = 1;
```

Comment:
I guess there are multiple different LENGTH metafunctions

Reply:
I.e. I should double-check if there are different function that still use the enums?

Reply:
Wo kein Ankläger, da ist kein Angeklagter. Not blocking for me.